Rampastring.Tools.nuspec

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package >
+  <metadata>
+    <id>devo1929.Rampastring.Tools</id>
```

Comment:
This would get changed before merging to main repo. This is package ID exactly as the library will appear in Nuget. Once it's deployed via a personal API key, that Nuget account then "owns" that package ID. However, there is a process in Nuget to transfer ownership.